src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -12764,10 +12764,7 @@ Unit* Creature::SelectVictim()
         return NULL;
 
     if (target && _IsTargetAcceptable(target) && CanCreatureAttack(target))
-    {
-        SetInFront(target);
         return target;
```

Comment:
Just tested it with a simple .cast back 11366, and it is not (spell was cast but serverside the npc didn't change his orientation until the cast was done).
But, if a player is tanking an NPC and NPC cast spell on another player, without this change it would have had his serverside orientation looking at the tank.